### DISCUSSION TITLE
Any plans to add indicator/strategy management?

### DISCUSSION THREAD

**Discussion by skwasha** [Mathieu2301/TradingView-API#56] [Ideas]
Are there any plans on the roadmap for adding the ability to manage access to indicators? As of now, this is a manual process on the TV platform (via Manage Access).

**Comment by Mathieu2301** [comment]
Hmm interesting request, I'll look into it.

**Comment by Mathieu2301** [comment]
I've tried [something](https://github.com/Mathieu2301/TradingView-API/blob/main/examples/PinePermManage.js), please tell me if it works :)
